### PR TITLE
Issue44: Episode rendering and export into csv

### PR DESCRIPTION
**Pull request by shamith2**
#44 
The episode will be rendered into gif or other video file, and agent's location will be exported into csv file if agent is alive

### CODE REVIEW

File: gym_cap/envs/cap_env.py [hunk 1]
```diff
@@ -524,7 +525,13 @@ def render(self, mode='human'):
                         [10+SCREEN_W//2, 10+SCREEN_H//2], [SCREEN_W//2-10, SCREEN_H//2-10])
         self._env_render(self._env,
                         [10, 10+SCREEN_H//2], [SCREEN_W//2-10, SCREEN_H//2-10])
-
+        self._env_render(self.blue_memory,
```
Comment (skim0119) [inline gym_cap/envs/cap_env.py]: Render memory when the memory_mode is either fog or full

**Review by skim0119 — CHANGES_REQUESTED** [review]
On render file, remove any statistics.

The script output some number of failure mode and success mode.
Save the video in 'render' folder, and save the corresponding script (in csv).

Only render full observation (env._env).

File: gym_cap/envs/cap_env.py (hunk 1 above)
Comment (shamith2) [reply gym_cap/envs/cap_env.py]: i updated cap_env (issue 38)

File: cap_render.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        pass
+    
+def export_data(filename):
+    """
+    agent's location will be exported into csv file.
+    If agent is dead, leave it static
+    
+    """
+    
+    if Agent.isAlive:
```
Comment (skim0119) [inline cap_render.py]:
```bash
pygame 1.9.4
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "cap_render.py", line 132, in <module>
    export_data(r'data.csv')
  File "cap_render.py", line 110, in export_data
    if Agent.isAlive:
AttributeError: type object 'Agent' has no attribute 'isAlive'
```

**Review by skim0119 — CHANGES_REQUESTED** [review]

**Comment by skim0119** [comment]
Feature moved into the branch 'render'

**Pull request closed without merging**